source/extensions/filters/network/dubbo_proxy/filters/well_known_names.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+
+#include "common/singleton/const_singleton.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+namespace DubboFilters {
+
+/**
+ * Well-known http filter names.
+ * NOTE: New filters should use the well known name: envoy.filters.dubbo.name.
+ */
+class DubboFilterNameValues {
```

Comment:
struct?

Reply:
class is used to keep the style consistent with other NameValues definitions